diff --git a/init.py b/init.py
index 93edcac..ad1f58d 100644
--- a/init.py
+++ b/init.py
@@ -68,14 +68,8 @@ CDockFocusController = ads.CDockFocusController
 CDockManager = ads.CDockManager
 CDockSplitter = ads.CDockSplitter
 CDockOverlay = ads.CDockOverlay
-CDockOverlayCross = ads.CDockOverlayCross
 CDockWidget = ads.CDockWidget
 CDockWidgetTab = ads.CDockWidgetTab
-CDockingStateReader = ads.CDockingStateReader
 CElidingLabel = ads.CElidingLabel
 CFloatingDockContainer = ads.CFloatingDockContainer
-CFloatingDragPreview = ads.CFloatingDragPreview
-IFloatingWidget = ads.IFloatingWidget
 CIconProvider = ads.CIconProvider
-CSpacerWidget = ads.CSpacerWidget
-CTitleBarButton = ads.CTitleBarButton
